### ISSUE TITLE
Breaking Change in release 4.0.13 'The engine "node" is incompatible with this module.'

### ISSUE THREAD

**Issue by danstarns** [Xunnamius/next-test-api-route-handler#1115]
My Node version is 22.4.1.

After upgrading from v 4.0.12 to [4.0.13](https://github.com/Xunnamius/next-test-api-route-handler/releases/tag/v4.0.13) I get the error: 

```
error next-test-api-route-handler@4.0.13: The engine "node" is incompatible with this module. Expected version "^18.20.0 || ^20.15.0 || >=22.5.0". Got "22.4.1"
```

@Xunnamius the version is quite specific, does this need to be the case: 

```
^18.20.0 || ^20.15.0 || >=22.5.0
```

?

**Comment by Xunnamius** [comment]
Good catch. It's coming from browserslist, but I usually only bump engine with majors. Revert incoming.

**Comment by danstarns** [comment]
> 🎉 This issue has been resolved in version 4.0.14 🎉
> 
> The release is available on:
> 
> * [npm package (@latest dist-tag)](https://www.npmjs.com/package/next-test-api-route-handler/v/4.0.14)
> * [GitHub release](https://github.com/Xunnamius/next-test-api-route-handler/releases/tag/v4.0.14)
> 
> Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot 📦🚀

Thank you.
